.github/workflows/build-and-publish-images.yml

```
@@ -31,10 +31,19 @@ jobs:
       orchestrator_image: ${{ steps.meta-orchestrator.outputs.tags }}
     
     steps:
+    # Option 1: GitHub App Token (Recommended)
+    - name: Generate GitHub App Token
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)